remediation/fence-agents-remediation/fence-agents-remediation.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: default
+title: Fence Agents Remediation
+nav_order: 4
+has_children: true
+parent: Remediation
+---
+
+## Fence Agents Remediation
+
+Generally Available
+{: .label .label-green }
+
+The [Fence Agents Remediation](https://github.com/medik8s/fence-agents-remediation#readme) (FAR) is a Kubernetes operator that uses external tools to *fence* unhealthy nodes.
+The tools are a set of well-known [upstream fence agents](https://github.com/ClusterLabs/fence-agents), where each one of them can be used for different environments,
+to fence a node using a traditional Application Programming Interface (API) call that reboots a node.
+By doing so FAR can minimize downtime for stateful applications, restore compute capacity if transient failures occur, and increase the availability of workloads.
+
+FAR is not only fencing a node when it becomes unhealthy, it also tries to *remediate* the node from being unhealthy to healthy again.
+It adds a taint to evict stateless pods, fence the node with a fence agent, and after reboot it completes the remediation with
+resource deletion to remove any remaining workloads (mostly stateful workloads).
+
+Similar to the other [remediation operators from Medik8s](https://www.medik8s.io/remediation/remediation/#implementations), FAR can be used to remediate a node that has been detected/selected to be remediated.
```

Comment:
> FAR can be used to remediate a node that has been detected/selected to be remediated

It sounds a bit complicated to read to me. The core is that it can work both standalone or in conjunction with another tool, in a more automatic fashion, right?

Reply:
Correct

Reply:
So I think would be easier to understand if saying that directly, like

"Far, similarly ..., can be used standalone, manually creating a CR, or together with another operator that automatically detects/select the target node and create the CR via a template"

or something like that

Reply:
>the core is that it can work both standalone or in conjunction with another tool,

The paragraph is to introduce the option of using another tool with FAR. I eventually added the "where to find FAR in the community" senetence, since I thought it could align here.
>So I think would be easier to understand if saying that directly

In the next sentence, we mention how FAR can work with NHC, another detecting mechanism (not necessarily an operator) or alone but with the use of an admin. I fixed to sentence tagged sentence.

Reply:
I think it is still unclear that this selection/detection process is automatic.

> The selection can be done with the Node HealthCheck Operator

while it might be clearer if re-phrased somehow like

> The selection can be made by NHC, which automatically detects... and creates the CR for...

Although, "selection" does not sound right to me. Maybe just "detected unhealthy".

Reply:
>>while it might be clearer if re-phrased somehow like

>The selection can be made by NHC, which automatically detects... and creates the CR for...

But this way the detection seems automatic and not the far cr creation. Both are important, but only the far CR creation that can happen automatically is important for FAR POV.
The paragraph is not about NHC working automatically rather FAR can work automatically using another tool/operator.
ATM the creation is mentioned to happen automatically, but maybe it was not stressed out properly being written in parenthesis. Is that what bugs you?

Reply:
>Although, "selection" does not sound right to me. Maybe just "detected unhealthy".

 remediates a node ... for the unhealthy detected node.
The detection can be done ...

Reply:
>selection/detection process is automatic.

The detection by NHC is not mentioned to be automatic but the far CR creation does (see below).
>which detects the node’s health (and automatically creates far CR)

I think the latter is more important and related to FAR. Do you still think it is important? 

>Although, "selection" does not sound right to me. Maybe just "detected unhealthy".

I changed it a little bit.